frontend/src/component/project/Project/ProjectSettings/ProjectActions/ProjectActionsTable/ProjectActionsActionsCell.tsx

```diff
@@ -38,21 +38,25 @@ export const ProjectActionsActionsCell = ({
             <TooltipLink
                 tooltip={
                     <StyledActionItems>
-                        {actions.map(({ id, action, executionParams }) => (
-                            <div key={id}>
-                                <strong>{action}</strong>
-                                <StyledParameterList>
-                                    {Object.entries(executionParams).map(
-                                        ([param, value]) => (
-                                            <li key={param}>
-                                                <strong>{param}</strong>:{' '}
-                                                {value}
-                                            </li>
-                                        ),
-                                    )}
-                                </StyledParameterList>
-                            </div>
-                        ))}
+                        {actions.map(
+                            ({ action, executionParams, sortOrder }) => (
+                                <div
+                                    key={`${actionSetId}/${sortOrder}_${action}`}
```

Comment:
Do we need this change? When we render `ProjectActionsActionsCell.tsx` all actions have a unique DB-assigned id, right?

Reply:
Eventually, yes, but we also need to change the request body schema, cause sending an id yields an error (at least in dev)